Source/WebCore/editing/FrameSelection.cpp

```diff
@@ -2353,7 +2353,7 @@ void FrameSelection::setFocusedElementIfNeeded()
             // so add the !isFrameElement check here. There's probably a better way to make this
             // work in the long term, but this is the safest fix at this time.
             if (target->isMouseFocusable() && !isFrameElement(target)) {
-                CheckedRef(m_document->page()->focusController())->setFocusedElement(target, *m_document->frame());
+                CheckedRef(m_document->page()->focusController())->setFocusedElement(target, *m_document->frame(), { { }, { }, { }, FocusTrigger::Bindings, { } });
```

Comment:
This is not bindings. Why are we using FocusTrigger::Bindings?

Reply:
We should probably do that. But more importantly, why is not resetting m_latestFocusTrigger right for all calls of this function? This function is called whenever selection is changed. If the problem here lies when this function is called by setSelectionRange then we need a more targeted code change instead.